pkg/provision/providers/docker/docker.go

```diff
@@ -48,19 +48,26 @@ func (p *provisioner) GenOptions(networkReq provision.NetworkRequest) []generate
 		generate.WithPersist(false),
 	}
 
+	networkConfig := &v1alpha1.NetworkConfig{
+		NetworkInterfaces: []*v1alpha1.Device{
+			{
+				DeviceInterface: "eth0",
+				DeviceIgnore:    true,
+			},
+		},
+	}
```

Comment:
I am thinking that skipping DHCP might be the better approach. Reason is that `eth0` isn't guaranteed to be the interface name. Also, if a user has more than one interface, we still get the DHCP delay.

Reply:
this is docker, I guess there's nothing more than `eth0`

and skipping DHCP is not an option, as config doesn't validate ("invalid addressing method")

Reply:
Just realized that my comment would be true only if the container was running with host net enable. Ignore me.